node/node.go

```diff
@@ -1182,7 +1182,11 @@ func (node *AlgorandFullNode) StartCatchup(catchpoint string) error {
 		node.log.Warnf("unable to create catchpoint catchup service : %v", err)
 		return err
 	}
-	node.catchpointCatchupService.Start(node.ctx)
+	err = node.catchpointCatchupService.Start(node.ctx)
+	if err != nil {
+		node.log.Warnf(err.Error())
+		return MakeCatchpointNoPeersFoundError(catchpoint)
```

Comment:
nit: this error seems too specific for this level of the code, there could be other errors now (i.e. a bad catchpoint label) or new conditions added in the future. I'd suggest creating the error type inside catchupservice or making the error type more generic, such as `UnableToStartCatchup`.